istioctl/pkg/waypoint/waypoint.go

```diff
@@ -235,6 +242,10 @@ func Cmd(ctx cli.Context) *cobra.Command {
 					fmt.Fprintf(cmd.OutOrStdout(), "Warning: namespace is not enrolled in ambient. Consider running\t"+
 						"`"+"kubectl label namespace %s istio.io/dataplane-mode=ambient"+"`\n", ns)
 				}
+				if hasSidecarInjection {
+					fmt.Fprintf(cmd.OutOrStdout(), "Warning: namespace is enrolled in sidecar mode. It takes precedence over ambient mode. Consider running\t"+
+						"`"+"kubectl label namespace %s istio-injection-"+"`\n", ns)
```

Comment:
```suggestion
						"`"+"kubectl label namespace %s istio-injection=disabled"+"`\n", ns)
```

I believe we have to explicitly use `disabled` here, because of the implicit injection semantics of `istio.io/rev`

https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy